pkg/strategy/streaming_test.go

```diff
@@ -131,11 +119,12 @@ func TestStreamingOptionsArePassed(t *testing.T) {
 
 func TestDelegateStreamingDepedencies(t *testing.T) {
```

Comment:
minor: same as in other tests - TestDelegateStreamingDependencies spelling and in test